assets/js/applicationinsights.js

```diff
@@ -8,6 +8,16 @@ window.applicationInsights = (function () {
         const clickPluginInstance = new Microsoft.ApplicationInsights.ClickAnalyticsPlugin()
         const clickPluginConfig = {
           autoCapture: true,
+          callback: {
+            contentName: function (element) {
+              // If there is a id, use this as the content name
+              if (element.dataset.id) return element.dataset.id
```

Comment:
Annoyingly although the `data-id` updated the event name, I noticed the default `contentName` was still being populated in the event properties. This fixes that.